docs/Quickstart.md

```diff
@@ -1,114 +1,101 @@
 # Quickstart
 
-> To set up development machines manually instead of using the IoT Edge Dev Container, please see the [Manual Development Machine Setup Wiki](https://github.com/Azure/iotedgedev/wiki/manual-dev-machine-setup).
-
 This quickstart will run a container, create a solution, setup Azure resources, build and deploy modules to your device, setup and start the IoT Edge simulator, monitor messages flowing into IoT Hub, and finally deploy to the IoT Edge runtime.
 
-Using IoT Edge Dev Container is the absolute fastest way to get IoT Edge Dev Tool (aka `iotedgedev`) running on your dev machine.
-
-You can use the IoT Edge Dev Tool container to avoid having to install all the dependencies on your local dev machine. The only thing you need to install is Docker. All of the other dev dependencies are included in the container.
-
-<!-- Here's a 3 minute video walk-through of this Quickstart:
-
-[![Azure IoT Edge Dev Tool: Quickstart](assets/edgedevtoolquickstartsmall.png)](https://aka.ms/iotedgedevquickstart) -->
+At the end of this quickstart, you should have the following:
 
-The only thing you need to install is Docker. All of the other dev dependencies are included in the container.
+- Development environment.
+- IoT Hub deployed in your Azure subscription.
+- A directory with the [IoT edge deployment manifest](https://docs.microsoft.com/en-us/azure/iot-edge/module-composition?view=iotedge-2020-11), [.env file](.env.tmp) with the environment variables needed for the `iotedgedev` tool, and the source file of a custom IoT edge module (`filtermodule`).
+- The docker image of the `filtermodule`.
+- Starting the IoT Edge simulator, which runs the containers defined in the device manifest.
+- Monitoring the messages sent to the IoT Hub.
+- Cleaning up the docker containers and images.
 
-1. **Install [Docker CE](https://docs.docker.com/install/)**
+## Pre-requisites
 
-    - For Windows, please follow [the document](https://docs.docker.com/docker-for-windows/#shared-drives) to open Docker Settings and setup a Shared Drive.
-    - For macOS, please follow [the document](https://docs.docker.com/docker-for-mac/#file-sharing) to choose local directories to share with your containers.
+- Docker (see [Install Docker docs](Install-Docker.md) for details) and add user to `docker` Unix group (see [documentation](https://docs.docker.com/engine/install/linux-postinstall/)).
```

Comment:
Nit-picky comment: Does this really need to be in it's own file? It's only a paragraph, which makes it a bit annoying to have to checkout. I'd understand if this section is used in multiple different files, but right now it's just one place

Reply:
The reason I put in a separate file is because I don't consider it to be essential here. It's just a small manual on how to install docker in different machines, which I don't think belongs in the main readme.